### ISSUE TITLE
Ticking Crash

### ISSUE THREAD

**Issue by D3nnis3n** [BuildCraft/BuildCraft#4145]
BuildCraft version:  7.99.16
Forge version: 2673
Link to crash report or log: https://gist.github.com/D3nnis3n/02972a418660d63f4337b37323cab43b
Singleplayer or multiplayer: Multiplayer
Steps to reproduce: Start server, login near the mentioned block, server crashes
Additional information:

**Comment by AlexIIL** [comment]
```
java.lang.ClassCastException: mcmultipart.util.MCMPWorldWrapper cannot be cast to net.minecraft.world.WorldServer

at noppes.npcs.api.wrapper.EntityWrapper.<init>(EntityWrapper.java:120)
```

I don't think this is buildcraft itself crashing - do you know what mod contains the  `noppes.npcs` package? (The mod name will probably be similar to the package name)

**Comment by D3nnis3n** [comment]
Custom NPCs - which we do not use at all. It's just installed.

**Comment by AlexIIL** [comment]
Ok, can you try removing it?

**Comment by D3nnis3n** [comment]
I can't reproduce the issue. The pipes at that place have been placed for three hours. Just some time they crashed. I'll remove it - but i can't know if it worked, as this issue does happen out of nothing.

**Comment by AlexIIL** [comment]
Does it generate *exactly* the same crash report every time?

**Comment by D3nnis3n** [comment]
Yes, but it's another block everytime. The Block Coordinates do change. We removed around 12 fluid pipes, then it was okay.

**Comment by AlexIIL** [comment]
Hmmm, that means that the `noppes.npcs.api.` package is still present, and probably contained by a mod.
